test/mason/publish-tests/noFork.prediff

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python3
```

Comment:
I haven't seen any other tests require python3, so lets keep it python here. I assume we'll convert them all at once, whenever we get around to that.